sdk/rust/src/signing/secp256k1.rs

```diff
@@ -151,6 +151,13 @@ impl Context for Secp256k1Context {
             Ok(pk) => Ok(Box::new(pk))
         }
     }
+
+    fn new_random_private_key(&self) -> Result<Box<PrivateKey>, Error> {
```

Comment:
Any reason this shouldn't be Secp256k1PrivateKey::new_random()?

Reply:
It looks like the other SDKs provide both

Reply:
Decided to add `Secp256k1PrivateKey::new_random()` later.